app/renderer/src/reducers/copiedToClipboard.js

```diff
@@ -0,0 +1,21 @@
+import {
+  UPDATE_QUERY,
+  SELECT_ITEM,
+  SELECT_NEXT_ITEM,
+  SELECT_PREVIOUS_ITEM,
+  COPY_ITEM,
+} from '../actions/types';
+
+export default function (state = false, { type }) {
+  switch (type) {
+    case UPDATE_QUERY:
```

Comment:
We should reset on details-pane actions.

- RESET_DETAILS
- CLOSE_DETAILS
- OPEN_DETAILS

Also:

- RESET_QUERY
- RESET_RESULTS -